tasks/first_server_restore.yml

```diff
@@ -0,0 +1,18 @@
+- name: Restore etcd - get all nodes
+  ansible.builtin.shell: |
+    set -o pipefail
```

Comment:
pipefail is not needed here

Reply:
Oh, good catch, copied over the merge conflict change without thinking.